tensorflow/compiler/tf2xla/functionalize_cond.cc

```diff
@@ -254,7 +257,8 @@ StateMap::AncestorId StateMap::GetAncestorId(
 }
 
 void StateMap::ResetAncestorId(const Node* node, StateMap::AncestorId id) {
-  if (node->id() < node_to_ancestorid_map_.size())
+  const int64 node_to_ancestorid_map_size = node_to_ancestorid_map_.size();
+  if (node->id() < node_to_ancestorid_map_size)
```

Comment:
Same as above. These locals are short lived, they are declared immediately before they are used so we don't need to be overly verbose.

Reply:
Don't forget about these ones.